src/features/controls.ts

```diff
@@ -290,18 +287,38 @@ export const ControlsMixin = <T extends Constructor<ModelViewerElementBase>>(
           // again for this particular <model-element> instance:
           this[$userPromptedOnce] = true;
           this[$waitingToPromptUser] = false;
+          this[$promptElementVisible] = true;
 
           this[$promptElement].classList.add('visible');
         }
       }
 
+      if (this[$promptElementVisible]) {
+        const {
+          left: modelViewerLeft,
+          width: modelViewerWidth,
+        } = this.getBoundingClientRect();
+        const {
+          left: promptLeft,
+          width: promptWidth,
+        } = this[$promptElementSVG].getBoundingClientRect();
+
+        const modelViewerCenter = modelViewerLeft + (modelViewerWidth / 2);
+        const promptCenter = promptLeft + (promptWidth / 2);
+        const promptOffsetRelative = (promptCenter - modelViewerCenter) / modelViewerWidth;
```

Comment:
We need to guard against when modelViewerWidth = 0 to avoid divide by zero errors.

This can happen if the model viewer starts hidden on a page.

Reply:
Good call! Fixed 👍